recipe/meta.yaml

```diff
@@ -19,13 +19,14 @@ requirements:
     - cmake
     - swig
     - pkgconfig
-    - boost 1.64.*
+    - boost 1.67.*
     - expat 2.2.*
     - glib
     - pango 1.40.*
     - proj4 4.9.3
     - python-eccodes
     - python
+    - jinja2
     - numpy 1.8.*
     - perl 5.22.2.1
     - perl-xml-parser
```

Comment:
Thanks, @kynan! Magics++ does not need `perl-xml-parser` any more, it seems --- and possibly `perl` can be dropped, too (looks like we're using Jinja2 templates instead of XML nowadays).

Reply:
Feel free to make those changes @carletes. You're a maintainer, so you can also push to the bot's branch. There are failing tests now, which I won't have time to debug.

Reply:
Hi @kynan & @carletes ,
just a quick note to say that I work already in a fork an a revised version of the scripts and hope soon to make a pull request. Much has changed, especially on the dependencies.

Reply:
@kynan: Will do, thanks for your help! Quick question: how do we do so that @StephanSiemen becomes a maintainer as well?